app/controllers/api/v1/reservations_controller.rb

```diff
@@ -1,2 +1,13 @@
 class API::V1::ReservationsController < ApplicationController
+  include CurrentUserConcern
+
+  def index
+    @user_reservation = Reservation.all
+
+    if @user_reservation
+      render json: @user_reservation
+    else
+      render json: { status: 401 }
+    end
```

Comment:
This is not necessary, since the exceptions are being handled by the Application controller.